facade/src/main/java/com/redhat/lightblue/migrator/facade/ServiceFacade.java

```diff
@@ -357,4 +397,19 @@ public TimeoutConfiguration getTimeoutConfiguration() {
     public void setTimeoutConfiguration(TimeoutConfiguration timeoutConfiguration) {
         this.timeoutConfiguration = timeoutConfiguration;
     }
+
+    /**
+     * Shutdown thread pool executor for all facades.
+     *
+     */
+    public static void shutdown() {
```

Comment:
I'm not sure if we need to worry about this, since we restart app server after every change. It's not that much of a problem if there is a leak during hot deployment and we know how to fix it (a singleton to shutdown facade on pre-destroy) if we decide to do so.